docs/overview.md

```diff
@@ -0,0 +1,64 @@
+Interledger routes packets of value from source to destination. Computers on the Interledger network are 
+called *nodes*. Nodes can take one or more of the following roles: 
+* Sender – Initiates a value transfer.
+* Router – Applies currency exchange and forwards packets of value. This is the intermediary node between 
+the sender and the receiver. 
+* Receiver – Receives the value.
+
+![ILP-nodes](images/ilp-nodes.png)
+
+## How does Interledger work?
+At the core of Interledger is the [Interledger Protocol (ILPv4)](https://interledger.org/rfcs/0027-interledger-protocol-4/), 
+which is a set of rules that defines how nodes should send value over the Interledger network. ILPv4 is a *request/response* 
+protocol, where each request or response is sent or received in the form of ILP packets. Typically, a single aggregate 
+payment from source to destination is split into multiple ILP packets. Each packet contains transaction 
+information, which is private to the participating nodes. ILPv4 has three packet types -  *Prepare*, *Fulfill*, and *Reject*. 
+
+![ILP-packets](images/ilp-packets.png)
+
+The sender constructs and sends a Prepare packet as a request to the connecting router. The routers forward the packet 
+until it reaches the receiver. The receiver then accepts or rejects the packet by sending a Fulfill packet or a 
+Reject packet as the response. The routers relay the response from the receiver back to the sender. When the sender 
+receives a Fulfill packet, it knows that the packet was successfully delivered to the receiver. The sender then 
+continues to send the remaining Prepare packets until the value is fully transferred. 
+
+Interledger does not rely on any single payment network for processing value transactions. You can connect with 
+an Interledger node at any time to join the network. Furthermore, Interledger sends value as tiny data packets, 
+which makes transactions fast, secure, and inexpensive.
+
+## Building on Interledger
+Build payments into your apps or other protocols without tying yourself to a specific currency or payment network. 
+Create accounts on our demo ledgers and start sending Interledger payments with the client libraries. 
+Check out [Getting Started](https://interledger.org/docs/tutorials/getting-started.html) for more details.
+
+## Interledger architecture
+Interledger enables payments across many different types of ledgers. The Interledger Protocol Suite is comprised of 
+four layers: the Application, Transport, Interledger, and Ledger protocols. To learn more, see the Interledger 
+[Architecture Overview](https://interledger.org/rfcs/0001-interledger-architecture/).
+
+## Protocol specs and APIs
+To dive into the technical specs, see the [Interledger RFCs](https://github.com/interledger/rfcs). Also see the documentation for the components of the 
+reference implementation.
+
+## Security
+Interledger provides secure multi-hop payments using 
```

Comment:
It might be good to mention that when we say secure we mean that money cannot be lost or stolen by intermediaries

Reply:
As discussed, maybe it's ok to leave it as is.